test/indexer/test_indexer.py

```diff
@@ -173,8 +201,9 @@ def test_deletion(self):
                                        metadata_files=bundle.metadata_files)
                     self._index_bundle(bundle)
                     hits = self._get_all_hits()
-                    self.assertEqual(len(hits), size * 2)
-                    num_aggregates, num_contribs = 0, 0
+                    # -1 because there is no replica for the bundle entity
+                    self.assertEqual(len(hits), size * 3 - 1)
```

Comment:
How about `len(DocumentType)` instead of the magic number `3` here?

Reply:
Hmm. Interesting idea but I'm not convinced it's appropriate. If we extend `DocumentType` to include more members in the future, we won't necessarily wish to count the new members in this context. The indexer tests are replete with these kinds of "magic" numbers where we count the number of hits, so I believe this is also consistent with precedent.